helper/staking/staking.go

```diff
@@ -173,6 +205,16 @@ func PredeployStakingSC(
 			types.StringToHash(hex.EncodeUint64(uint64(indx + 1)))
 	}
 
+	// Set the value for the minimum and maximum number of validators
+	min := PadLeftOrTrim(minNumValidators.Bytes(), 4)
+	max := PadLeftOrTrim(maxNumValidators.Bytes(), 4)
+	storageMap[types.BytesToHash(big.NewInt(minAndMaxNumValidator).Bytes())] = types.BytesToHash(
+		append(
+			max,
+			min...,
+		),
+	)
```

Comment:
This padding, and combining later on in the append can be a separate method, as it's universal for bytes that are packed into a  single uint256

Reply:
Not sure what do you mean with this, please explain

Reply:
I think the code looks much cleaner now. Fixed 79db141a49442b1de0118c5fb7c588d0e7775918